src/web/lib/getCommentContext.ts

```diff
@@ -22,17 +22,77 @@ type CommentContextType = {
     discussionKey: string;
     discussionWebUrl: string;
     discussionApiUrl: string;
-    orderBy: 'oldest' | 'newest' | 'mostrecommended';
-    pageSize: 20 | 25 | 50 | 100; // TODO: Review these https://trello.com/c/7v4VDNY0/1326-review-page-size-values
```

Comment:
We no longer need the 20 option because we can force the use of 25 by passing it in the request